blog/docs/releases/announcing-knative-v1-5-release.txt

```diff
@@ -0,0 +1,205 @@
+---
+title: "1.5 release"
+linkTitle: "1.5 release"
+Author: "[dprotaso](https://twitter.com/lintinmybelly)"
+Author handle: https://github.com/dprotaso
+date: 06-06-2022
+description: "Knative 1.5  release announcement"
+type: "blog"
+---
+
+
+## Announcing Knative 1.5 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in [Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+## Table of Contents
+- [Highlights](#highlights)
+- [Serving](#serving)
+- [Eventing](#eventing)
+- [Eventing Extensions](#eventing-extensions)
+    - [Apache Kafka Broker](#apache-kafka-broker)
+    - [RabbitMQ Broker and Source](#rabbitmq-broker-and-source)
+- `kn` [CLI](#client)
```

Comment:
```suggestion
- [`kn` CLI](#client)
```

Reply:
(looks like the link has always been on just the CLI part, but in a vacuum I'd say it's better to link the whole line)